linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpLoggerConfig.scala

```diff
@@ -0,0 +1,34 @@
+package io.buoyant.linkerd.protocol
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.{Filter, Service, ServiceFactory, Stack, Stackable}
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.util.Future
+import io.buoyant.config.PolymorphicConfig
+
+abstract class HttpLoggerConfig extends PolymorphicConfig {
+  @JsonIgnore
+  def mk(params: Stack.Params): Filter[Request, Response, Request, Response]
+}
+
+object HttpLoggerConfig {
+  object param {
+    case class Logger(logger: (Stack.Params) => Filter[Request, Response, Request, Response])
```

Comment:
Spent some time with @adleong trying to implement `override def router(params: Stack.Params): Router`. At this point in the code we don't have access to the stack without plumbing it through a few layers. Is this change warranted? Are there better approaches?